test/types/store/descriptor.test.ts

```diff
@@ -0,0 +1,91 @@
+// This test is not intended to be run by JavaScript.
```

Comment:
As the convention is to add test files into separate paths, names don't need a `test` suffix.